iotile_ext_cloud/iotile/cloud/cloud.py

```diff
@@ -91,6 +92,46 @@ def device_info(self, device_id):
 
         return dev
 
+    @param("fleet_id", "integer", desc="Id of the fleet we want to retrieve")
+    @return_type("basic_dict")
+    def get_fleet(self, fleet_id):
+        """ Returns the devices in the given fleet"""
+        api = self.api
+        if (fleet_id > pow(16,12) or fleet_id < 0):
+            raise ArgumentError("Fleet id outside of bounds")
+        slug = "g--" + device_id_to_slug(fleet_id)[8:]
+        try :
+            return api.fleet(slug).devices.get()
```

Comment:
I would have this return a dictionary mapping device slugs to devices to make this function easier to work with.  This will simplify the `get_whitelist` function

```
results = api.fleet(slug).devices.get()
entries = results.get('results', [])

return {entry['device']: entry for entry in entries}
```